server/gokbg3/grails-app/controllers/org/gokb/rest/TippController.groovy

```diff
@@ -188,9 +189,10 @@ class TippController {
         if (tipp_validation.valid) {
           def jsonMap = obj.jsonMapping
 
-          obj = TitleInstancePackagePlatform.upsertDTO(reqBody, user)
+          obj = restMappingService.updateObject(obj, jsonMap, reqBody)
```

Comment:
ich kann gerade nicht ausmachen, welche Version von Dir kam...

Reply:
Both should be fine so you can leave it like this.